### PR TITLE
New biz lines

### PR DESCRIPTION
**Pull request by ominina**
Made business line pages and deleted infrastructure.

### CODE REVIEW

**Comment by wslack** [comment]
1. Permalinks need to be prefaced with "business lines" in the URL like the Infrastructure page was
2. URLs need hyphens (no spaces) like https://github.com/18F/intake/blob/18f-pages-staging/pages/funding-and-iaa.md

**Comment by mbland** [comment]
I'll push fixes for these, but actually, under the new rules, you only need a `permalink:` for the home page now.

**Comment by wslack** [comment]
@ominina I think you have the baton to fix these; is that right?

**Comment by ominina** [comment]
I made the two fixes

**Comment by wslack** [comment]
Remove capitalizations in URLs please!

**Comment by wslack** [comment]
Oh, you also don't need the titles in your pages - the front matter will take of it.

File: pages/Acquisition-Services.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+Permalink: /business-lines/Acquisition-Services/
+Title: Acquisition Services 
+---
+
+## Acquisition Services
```
Comment (wslack) [inline pages/Acquisition-Services.md]: Example!

**Comment by ominina** [comment]
all edited!

**Pull request merged**